src/mlpack/tests/main_tests/pca_test.cpp

```diff
@@ -96,23 +93,22 @@ BOOST_AUTO_TEST_CASE(PCANoVarRetainTest)
   mlpackMain();
 
   // Check that the output has 1 dimensions.
-  BOOST_REQUIRE_EQUAL(IO::GetParam<arma::mat>("output").n_rows, 1);
-  BOOST_REQUIRE_EQUAL(IO::GetParam<arma::mat>("output").n_cols, 5);
+  REQUIRE(IO::GetParam<arma::mat>("output").n_rows == 1);
+  REQUIRE(IO::GetParam<arma::mat>("output").n_cols == 5);
 }
 
 /**
  * Check that we can't specify an invalid new dimensionality.
  */
-BOOST_AUTO_TEST_CASE(PCATooHighNewDimensionalityTest)
+TEST_CASE_METHOD(PCATestFixture, "PCATooHighNewDimensionalityTest",
```

Comment:
I think there is no style guideline for this, but in the other tests the second line aligns with the first argument from the first one.

https://github.com/mlpack/mlpack/blob/6ced8d236590c86249d66204d8c1991beeb61249/src/mlpack/tests/main_tests/kmeans_test.cpp#L52-L53

is one example.